shared/studio/tabs/auth/index.tsx

```diff
@@ -278,6 +278,54 @@ const ConfigPage = observer(function ConfigPage() {
             </div>
           </div>
         </div>
+
+        <div className={styles.gridItem}>
+          <div className={styles.configName}>allowed_redirect_urls</div>
+          <div className={styles.configInputWrapper}>
+            <div className={styles.configInput}>
+              {state.configData ? (
+                <>
+                  <TextArea
+                    value={
+                      state.draftAllowedRedirectUrls.value ??
+                      state.configData.allowed_redirect_urls
+                    }
+                    onChange={(urls) => state.draftAllowedRedirectUrls.setValue(urls)}
+                    error={state.draftAllowedRedirectUrls.error}
+                    size={32.5}
+                  />
+                  {state.draftAllowedRedirectUrls.value != null ? (
+                    <>
+                      <Button
+                        className={styles.button}
+                        label={
+                          state.draftTokenTime.updating ? "Updating" : "Update"
```

Comment:
```suggestion
                          state.draftAllowedRedirectUrls.updating ? "Updating" : "Update"
```

Reply:
Oops! That's what I get for not using find-replace. Nice catch!